lib/symboldatabase.cpp

```diff
@@ -8033,7 +8033,9 @@ std::string ValueType::str() const
         ret += " &";
     else if (reference == Reference::RValue)
         ret += " &&";
-    return ret.empty() ? ret : ret.substr(1);
```

Comment:
Variables in ternaries are not moved on return so I split the statement instead of introducing an explicit move.